index.bs

```diff
@@ -1253,7 +1253,7 @@ Technical Architecture Group Appointments</h5>
 	The [=Team=]'s choice of appointee(s)
 	is subject to ratification by secret ballot
 	by both the [=AB=] and the [=TAG=],
-	each requiring a two-thirds approval.
+	requiring a two-thirds approval from each of these groups counted separately.
```

Comment:
`from each of these groups` already covers `counted separately`.
```suggestion
	requiring a two-thirds approval from each of these groups.
```

Reply:
As there was confusion about this, I think it is better to be explicit, and Florian's original wording is better.

Reply:
```suggestion
	requiring a two-thirds approval from each of these groups, counted separately.
```